src/tools/pbs_upgrade_job.c

```diff
@@ -90,6 +90,76 @@
 /* From pbs_ifl.h */
 #define PBS_MAXSEQNUM_PRE19	7
 #define PBS_MAXSVRJOBID_PRE19	(PBS_MAXSEQNUM_PRE19 - 1 + PBS_MAXSERVERNAME + PBS_MAXPORTNUM + 2)
+#define PBS_MAXSVRJOBID_19_20	(PBS_MAXSEQNUM - 1 + PBS_MAXSERVERNAME + PBS_MAXPORTNUM + 2)
+
+/*
+ * Replicate the jobfix structure as it was defined in 19.x and 20.x versions
+ */
+typedef struct jobfix_19_20 {
```

Comment:
Maybe rename the structure to jobfix_19_21 because it is same across all 3 versions?